### ISSUE TITLE
Needs to handle escaping properly

### ISSUE THREAD

**Issue by kadler** [IBM/ibmi-db2util#9]
Various file types may have issues when columns contain special characters. eg. if a column contains a `"` in a CSV, this will break the string quoting, resulting in a broken CSV. Likewise for JSON, though the escaping is different there.

Each format needs to have an escape function that adjusts the data prior to displaying.

**Comment by worksofliam** [comment]
@kadler What more is there to do? Looks ok from my shell, and I can see in the code that the basics are handled.

* JSON: https://github.com/IBM/ibmi-db2util/blame/master/format_json.c#L104
* CSV: https://github.com/IBM/ibmi-db2util/blob/master/format_csv.c#L29

```
-bash-5.1$ db2util "values ('hello \" world')"
"hello "" world"
-bash-5.1$ db2util -o json "values ('hello \" world')"
{"records":[
{"00001":"hello \" world"}
]}
```

**Comment by kadler** [comment]
Looks like you fixed it in #10 but the PR wasn't tied to this issue and it didn't get closed.
